content/applications/inventory_and_mrp/inventory/warehouses_storage/removal_strategies.rst

```diff
@@ -0,0 +1,15 @@
+:nosearch:
+
+==================
+Removal strategies
+==================
+
+.. toctree::
+   :titlesonly:
+
+   removal_strategies/removal
```

Comment:
I think you can actually transplant the contents of `removal.rst` into `removal_strategies` since it's purpose was to be a general/config doc for the other ones. If you think so too then can move the contents, delete the old file, and add a redirect to point here 👍

Reply:
good catch :)

Reply:
Applied the suggestion!